tests/test_matrix_utils_connect_rooms.py

```
@@ -0,0 +1,295 @@
+
+import mmrelay.matrix_utils as mx
+from mmrelay.matrix_utils import connect_matrix
+
+
+@patch("mmrelay.matrix_utils.matrix_client", None)
+@patch("mmrelay.matrix_utils.AsyncClient")
+@patch("mmrelay.matrix_utils.logger")
+@patch("mmrelay.matrix_utils.login_matrix_bot")
+@patch("mmrelay.matrix_utils.async_load_credentials", new_callable=AsyncMock)
+async def test_connect_matrix_alias_resolution_exception(
+    mock_load_credentials, mock_login_bot, _mock_logger, mock_async_client
```

Ruff (0.15.9)

[warning] Fixture `_mock_logger` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)